share/revng/test/tests/abi/build_abi_test_binaries_with_gcc.sh

```diff
@@ -31,12 +31,20 @@ mkdir -p "${OUTPUT_DIRECTORY}"
   -o "${OUTPUT_DIRECTORY}/functions"
 
 # Build the binary for runner to `mmap`
+"${TRIPLE}gcc" -c \
+  ${CFLAGS} -O3 -static \
+  -fno-zero-initialized-in-bss \
```

Comment:
What was this for?

Reply:
Because when compiling for arms, the compiler (or is it linker?) is very eager to use `bss` where it can. And the current test toolchain does not support it.

Reply:
Add a comment

Reply:
Done.